ansible/roles/config-ocp-cluster/tasks/pipelinerun-listener.yml

```diff
@@ -164,3 +164,18 @@
             name: config-defaults
           data:
             default-cloud-events-sink: "http://el-pipelinerun-listener.{{ pipelinerun_listener_namespace }}.svc.cluster.local:8080"
+
+    - name: Set default Tekton CloudEvents for Tekton Bundles
```

Comment:
The file already exists on the cluster with a bunch more settings. By adding it here you will override the cluster settings. Can you just flip those switches manually instead of overriding the whole file?

Reply:
Updated!